src/Sylius/Component/Core/Test/Services/DefaultChannelFactory.php

```diff
@@ -22,17 +21,16 @@
 /**
  * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
  */
-class DefaultFranceChannelFactory implements DefaultStoreDataInterface
+class DefaultChannelFactory implements DefaultStoreDataInterface
```

Comment:
I would still call it `DefaultFranceChannelFactory` since it creates french channel :)